### PR TITLE
Allows mutant races into head roles again

### PR DESCRIPTION
**Pull request by Time-Green**
It was changed a while back ,without most people knowing

I made this as both a PR and a place to discuss this, as alot of people are unhappy with it.

Anyway, I expect this to be extremely controversial. 

(I know it has to be edited by the host or something, but this is just it's thread)
:cl:
tweak: After 4 revolutions and PETA activists, everyone can apply for headrole again.
/:cl:

### CODE REVIEW

**Comment by matskuman5** [comment]
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

*breath*

OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Comment by drderp3635** [comment]
No, all this will lead to is the overall degradation of the stations society, we can not allow the station to be run by snowflake cabages, or Mr eats-the-child, and god help us if we have firesuit fabio for captain. Get this shit outta here.

**Comment by Time-Green** [comment]
You guys are all very rachist. Just pointing that out.

**Comment by Super3222** [comment]
No. Mutant races have advantages, and the disadvantage is that they don't get head positions.

**Comment by drderp3635** [comment]
I'm fully aware of that, but also consider that that the only people who have access to the ai shouldn't be people the ai can shoot down for coming into the chamber as a non-human

File: config/game_options.txt [hunk 1]
```diff
@@ -165,7 +165,7 @@ PROTECT_ROLES_FROM_ANTAGONIST
 #PROTECT_ASSISTANT_FROM_ANTAGONIST
 
 ## If non-human species are barred from joining as a head of staff
-#ENFORCE_HUMAN_AUTHORITY
+maks america grate egayn
```
Comment (Super3222) [inline config/game_options.txt:168]: Get this out of here.

Comment (Time-Green) [reply config/game_options.txt:168]: I cannot edit config files, it must be done from the host. Editing this would do nothing. It was originally already turned off. So might aswell do it like this.

And people that think mutants should not be in command roles are rachist, just like...

Comment (Super3222) [reply config/game_options.txt:168]: What was the point of this PR than.
Make an issue for it instead, and contact the HCs or The Council.
Don't make the code dirty. What happens when someone downloads it for their own personal reasons? It'll make us look bad.

Comment (Time-Green) [reply config/game_options.txt:168]: Issues don't really get enough attention. 
And I never meant for it to be merged, the PR that made these changes in the first place was not merged either, it was closed and editted without knowing.

I needed to add something so I can PR it.

"Then make it on the forums daaaaa"
I'll rather put down a link there to this page.

Comment (Super3222) [reply config/game_options.txt:168]: Fair enough. I'll pin a DNM.

**Comment by AsV9** [comment]
This should be a forum vote and not limited to github participants.

**Comment by Jc1469** [comment]
> No. Mutant races have advantages, and the disadvantage is that they don't get head positions.

Except they also have disadvantages on top of that.
But I still don't think they should be heads.  Where's the fun of spawning as Drake Lazarus, chief engineer, when I can spawn as Drake Lazarus, atmos tech, and then reroute disposals to the chief engineer's office.

**Comment by ghost** [comment]
If you meant this to be a discussion thread, here's the deal.

It wasn't done without 'most people knowing' and that claim is literally a good eighth of what lead to me quitting the staff team and being a gigantic asshole.

It was done and publicized. Repeatedly. I made the PR exactly as you did and basically pasted it in OOC, constantly telling people to go look, go vote, etc, have a say in upcoming server changes.

No controversy against it.

Eventually, when it was fine, I got ASV9 to do it.

It was done with my best attempt at making people know and the consistent claim that it was 'stealth done' or something is the claim that pissed me off to start with and started this whole debacle, because I did nothing BUT make it known. I pasted it in OOC. I started votes. I messaged people -on fucking BYOND- to go read the PR and make their comments or it'd get done, and then the moment it's merged i'm assailed with numerous people whining saying I did it behind their back.

Unfortunately, that's not the case.

Now!

Now that i've debunked the first part of your ridiculous claim that it wasn't a known change, let's move onto the actual change.

No.

There's no reason for this.

First off, nonhumans not being eligible for head roles is part of lore. Yes, lore, since we've basically taken /tg/'s lore as we have none of our own. It's lead to some fun instances, such as rev rounds being motivated by this exact reason, which is the closest we'll ever get to actual RP.

Second, and more evidently-

Why would you make nonhumans who can be killed by the AI for causing harm eligible for head of staff positions? The captain orders the HoS to execute a traitor, and the AI promptly burns the lizardman Captain on a door and has a borg space him and it'd be entirely valid and logical because the highest authority on the station is also nonhuman AND hostile to humans.

The sort of conflicts this could lead to would be ridiculous, up to and including a cyborg beating the lizard HoS to death because he was lasering a human nukeop and just. wouldn't. stop.

Hence, due to law conflicts it just doesn't make sense to make nonhumans heads, and no you don't get to say 'well make them human again' because votes showed people didn't want that.

Tl;dr I DON't LIKE

**Comment by ghost** [comment]
oh and phytosians don't have a downside and anyone who claims they do is full of shit

you take slightly more burn damage and you can't use mechs (nobody makes them generally and if you ahve access to mechs you ahve access to better) and in turn you heal automatically

big whoop

**Comment by Time-Green** [comment]
Phytosian downsides: You take massive burn damage, need 2 flashlights in maint to not die. And the heaviest one: You die extremely quick from being in anything. You cannot go into a locker for more then 1 minute and most of the time you get cryo'd you're stuck there for a while since you suffocate being inside it.

**Comment by Time-Green** [comment]
But yes, I am aware they need a nerf. I myself thought about removing the no-food deal. Since well, plants need more then just light and air to survive.
